paper/paper.md

```diff
@@ -0,0 +1,141 @@
+
+2. [cdp-backend](https://github.com/CouncilDataProject/cdp-backend): A Python package containing CDP's database schema definition, a file format for transcripts generated by speech-to-text algorithms, an infrastructure specification, and processing pipelines. This package currently contains an event gather and processing workflow that will parse event details, generate a transcript for the event using either the provided closed caption file, or using Google Speech-to-Text from the provided event video, and finally, generate and store event metadata (voting records, thumbnails, minutes items, etc.) This package additionally provides a workflow for generating a TF-IDF based event index for weighted term search. Processing workflows (and all tools in `cdp-backend`) are separate from any one CDP instance so that all CDP instances can be easily upgraded whenever there is a new version of `cdp-backend` released.
+
+3. [cdp-frontend](https://github.com/CouncilDataProject/cdp-frontend): A TypeScript library containing all frontend components and a compiled web application. The web application allows for simple data exploration and sharing, and as such, acts as a method for providing context for events, voting records, and more because it allows users to interactively explore the data produced by the backend pipelines. The web application (and all components in `cdp-frontend`) is separate from any single CDP instance so that all CDP instances can be easily upgraded whenever there is a new version of `cdp-frontend` released.
+
+## Cookiecutter and the Produced Repository
+
+`cookiecutter-cdp-deployment` will generate all necessary files for an entirely new CDP instance as well as additional setup documentation for the user to follow to fully complete the instance deployment process.
+
+Utilizing [GitHub Actions](https://github.com/features/actions) and [GitHub Pages](https://pages.github.com/), data processing and web hosting are entirely free as long as the user sets their instance's GitHub repository visibility to public. While the limited primary costs of CDP tools are:
+
+1. [Google Speech-to-Text](https://cloud.google.com/speech-to-text/) for transcript generation.
+2. [Firebase Cloud Firestore](https://firebase.google.com/docs/firestore/) for event metadata storage and access.
+3. [Firebase Storage](https://firebase.google.com/docs/storage) for file storage and access.
+
+![CDP Core Infrastructure Figure. Event Gather Pipeline: inputs, GitHub Action, Python Pipeline tasks, and storage. Event Index Pipeline: inputs, GitHub Action, Python n-gram generation, and storage. Web Application Building: inputs, GitHub Action, and GitHub Pages.\label{fig:core-infra}](./assets/cdp_core_infrastructure.png)
+
+CDP tools allow for decentralized control over the management and deployment of each CDP instance while producing a standardized open-access dataset for both research and for municipal transparency and accessibility.
```

Comment:
Is "While the limited ... for municipal transparency and accessibility." one sentence? Having two whiles in a sentence sounds awkward. I would change the second "while" to "and" or another conjunction word.

If it's not one sentence, "While the limited primary costs of CDP tools are:" doesn't make sense to me. I would just change it to "The limited primary costs of CDP tools are:"